pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/user/UserDao.java

```diff
@@ -79,10 +80,12 @@ public Optional<UserDto> get(long userId) {
     @Override
     public int create(UserDto userDto) {
         SimpleResult results = inTransaction((conn) -> {
-            SimpleResult execResult = new SimpleResult();
+            SimpleResult execResult = new SimpleResult(-1);
             try (PreparedStatement stmt = conn.prepareStatement(SQL_INSERT_USER, PreparedStatement.RETURN_GENERATED_KEYS)) {
                 stmt.setString(1, userDto.getName().orElse(""));
                 stmt.setString(2, userDto.getEmail().orElse(""));
+                stmt.setString(3, userDto.getPhoneNumber().orElse(""));
```

Comment:
curious why "" rather than null ?

Reply:
I am typically reluctant to change existing code given the low DSM code coverage, but I'll look more closely at this. This method is called sparingly, so the fan out from changes is low. Since email is required, that part is at least incorrect.